recipes-core/images/core-image-minimal.bbappend

```diff
@@ -0,0 +1 @@
+IMAGE_INSTALL_append = " packagegroup-prplmesh"
```

Comment:
Ugh... Putting prplmesh in core-image-minimal seems like a stretch, no?

Reply:
Yocto Poky contains several variations of `core-image-minimal` and `core-image-sato`.
`core-image-sato` is graphic oriented and is not an image for routers.

So we have three options:
1) add prplMesh into `core-image-minimal`
2) add recipe for new image which i guess will be equal to `core-image-minimal` + prplmesh  :-)
3) remove image bbappend at all and add into README.md the steps about adding
```
IMAGE_INSTALL_append = " packagegroup-prplmesh"
```
into `conf/local.conf`

By my opinion they are almost equivalent. Maybe 3rd one is more clear for this layer.